L1Trigger/L1TMuonOverlap/src/AngleConverter.cc

```diff
@@ -88,6 +88,20 @@ int etaKeyWG2Code(const CSCDetId& detId, uint16_t keyWG) {
   return etaCode;
 }
 
+
+int fixCscOffsetGeom(int offsetLoc) {
+  // fix for CSC feo dependance from GlobalTag
+
+  // dump of CSC offsets for MC global tag
+  std::vector<int> offCSC = { -154, -133, -17, -4, 4, 17, 133, 146, 154, 167, 

```

Comment:
@konec 
Please document where we get these magic numbers from,
or if possible what do these numbers correspond to.
Note: documentation of constant numbers is a requirement 
before merging code into CMSSW.